gptel-transient.el

```diff
@@ -752,7 +752,26 @@ Customize `gptel-directives' for task-specific prompts."
     :pad-keys t])
 
 ;; ** Prefix for selecting tools
+(defun gptel--toggle-all-tools ()
+  "Toggle the selection state of all tools in the transient menu.
 
+If any tool is currently selected, deselect all tools.
```

Comment:
> If any tool is currently selected, deselect all tools

This is not how the function is working -- when some tools are selected it selects all tools instead.

Reply:
Have corrected the doc. My intent was to turn them all on